engine/src/test/java/org/hibernate/search/test/dsl/SortDSLTest.java

```diff
@@ -673,7 +682,7 @@ public void set(String name, Object value, Document document, LuceneOptions luce
 				return;
 			}
 
-			luceneOptions.addNumericFieldToDocument( name, doubleValue, document );
+			document.add( new DoubleDocValuesField( name, doubleValue ) );
```

Comment:
Simply because there's no way to add a doc values field using luceneOptions. It seems to be designed that way, seeing the examples in the documentation.
I agree it's a bit weird, though. Another ticket, maybe?